testing/tests/test_api.py

```diff
@@ -15,6 +15,30 @@ def test_verification_api(self, testapp):
         assert 200 == ver_res.status_code
         assert ver_res.json['results'][0]['tamper'] > 0.5
 
+    @pytest.mark.usefixtures("testapp")
+    def test_fps_parameters(self, testapp):
+        # test verification with 0 fps
+        ver_res = testapp.post('/verify', json=dict(source='testing/tests/data/master_4s_1080.mp4', orchestratorID='1',
+                                                    renditions=[{'uri': 'testing/tests/data/master_4s_1080.mp4',
+                                                                 'frame_rate': 0}]))
+        assert 200 == ver_res.status_code
+        assert ver_res.json['results'][0]['tamper'] < 0.5
```

Comment:
I think we should check the `frame_rate` field in the API response which should be true indicating that the frame rate check passed.

Reply:
I don't think it should pass for 0 value. Currently, it would neither fail nor pass, will leave 0 there. Though, I think the entire design solution to have validation results overwrite the fields with same name is questionable.

Reply:
> I don't think it should pass for 0 value. Currently, it would neither fail nor pass, will leave 0 there.

Left some thoughts about this in [this comment](https://github.com/livepeer/verification-classifier/pull/121#discussion_r457778483). Let me know what you think.

>  I think the entire design solution to have validation results overwrite the fields with same name is questionable.

Yeah agreed that this could probably be cleaned up outside of this PR.

Reply:
Fixed.